src/Umbraco.Core/PropertyEditors/DataValueReferenceFactoryCollection.cs

```diff
@@ -8,5 +10,43 @@ public class DataValueReferenceFactoryCollection : BuilderCollectionBase<IDataVa
         public DataValueReferenceFactoryCollection(IEnumerable<IDataValueReferenceFactory> items)
             : base(items)
         { }
+
+        public IEnumerable<UmbracoEntityReference> GetAllReferences(PropertyCollection properties, PropertyEditorCollection propertyEditors)
+        {
+            var trackedRelations = new List<UmbracoEntityReference>();
+
+            foreach (var p in properties)
+            {
+                // Injecting propertyEditorCollection is causing LightInject to throw a Recursive Dependancy error
+                // Hence using Current.PropertyEditors
```

Comment:
comment can be removed

Reply:
Done in this commit 4d78a2c848d7399ddaa13257a7d13526205f5b31 @Shazwazza